PIPELINE_DEPLOYMENT_GUIDE.md

```
@@ -0,0 +1,244 @@
+
+1. **Subgraph must be working** - The subgraph must be properly indexed and working. Check the Goldsky dashboard to ensure there are no "Failed to transact block operations" errors.
+
+2. **PostgreSQL Secret** - You need a PostgreSQL secret configured in Goldsky:
+   ```bash
+   goldsky secret create --name POSTGRES_SECRET_CMJIQCIFZ0 --value '{
+     "type": "jdbc",
+     "protocol": "postgresql",
+     "host": "your-db-host.com",
+     "port": 5432,
+     "databaseName": "your-database",
+     "user": "your-user",
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)